.github/workflows/autotools.yml

```
@@ -14,7 +29,24 @@ jobs:
     - uses: actions/checkout@v2
+    # test configure and build
     - name: configure
-      run: ./configure
+      run: ./configure --prefix=$PORTAUDIO_INSTALL_DIR
     - name: make
       run: make
+    # test make install
+    - name: install
+      run: make install
+    - name: list install dirs (post-install)
+      run: ls $PORTAUDIO_INSTALL_DIR $PORTAUDIO_INSTALL_DIR/include $PORTAUDIO_INSTALL_DIR/lib
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:4: Double quote to prevent globbing and word splitting

(shellcheck)